data/filters/normalize/reviewers.lua

```diff
@@ -0,0 +1,167 @@
+local List = require 'pandoc.List'
```

Comment:
A lot of this is taken from `data/filters/normalize-author-names.lua` and from `data/filters/normalize/authors.lua`.  It would be better to `require` that file (or to factor out some of that code).

Tbh, I forgot why there are two files with similar names that effectively do one task. I should probably clean that up.